java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -34,12 +34,14 @@ public class KafkaConsumerConfig {
     private final String oauthTokenEndpointUri;
     private final String additionalConfig;
     private final String saslLoginCallbackClass = "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler";
+    private final String tracingSystem;
+    private static final String DEFAULT_TRACING_SYSTEM = null;
```

Comment:
Strings are null by default so you can remove `= null`

Reply:
Without initialising to null, I'm getting the following error - `Variable 'DEFAULT_TRACING_SYSTEM' might not have been initialized`